src/Bridge/Symfony/Resources/config/services.xml

```diff
@@ -21,6 +21,7 @@
             <argument>%sonata.exporter.writer.xml.main_element%</argument>
             <argument>%sonata.exporter.writer.xml.child_element%</argument>
         </service>
-        <service id="sonata.exporter.exporter" class="Exporter\Exporter" public="true"/>
+        <service id="Exporter\Exporter" autowire="true"/>
```

Comment:
Let's not change this, and not use autowiring in a bundle, that's not recommended.
```suggestion
<service id="sonata.exporter.exporter" class="Exporter\Exporter" public="true"/>
```